src/ruby/ext/symengine/extconf.rb

```diff
@@ -0,0 +1,22 @@
+require 'mkmf'
+
+#Gives the ability to easily use alternate compilers to build the extension
+RbConfig::MAKEFILE_CONFIG['CC'] = ENV['CC'] if ENV['CC']
+
+extension_name = 'symengine/symengine'
+
+#Check to see if the csympy library required to build this extension exists.
+#Typically, we would want to use libcsympy installed, including the header
+#files
+#unless have_library("libcsympy")
+#    raise "libcsympy not found"
+#end
+#How is libcsympy installed? Not able to find libcsympy
```

Comment:
It depends how you install symengine, but it is either `src/libsymengine.a` or `src/libsymengine.so`. After it gets installed, it will be in `lib/`.